deploy/openshift4/nsx-ipi-helper.yaml

```diff
@@ -0,0 +1,49 @@
+# nsx-ipi-helper DaemonSet
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nsx-ipi-helper
+  namespace: nsx-system-operator
+  labels:
+    tier: nsx-networking
+    component: nsx-ipi-helper
+    version: v1
+spec:
+  selector:
+    matchLabels:
+      tier: nsx-networking
+      component: nsx-ipi-helper
+      version: v1
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        tier: nsx-networking
+        component: nsx-ipi-helper
+        version: v1
+    spec:
+      hostNetwork: true
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
+        - key: node.kubernetes.io/not-ready
+          effect: NoSchedule
+        - key: node.kubernetes.io/unreachable
+          effect: NoSchedule
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      initContainers:
+        - name: nsx-ipi-helper
+          image: registry.access.redhat.com/ubi8/ubi:latest
+          imagePullPolicy: IfNotPresent
+          command: ["/bin/bash", "-c", "yum -y install iptables; echo Disabling the DNS in localhost; iptables -A INPUT -p udp --dport 53 -j DROP; iptables -S INPUT; while ! curl localhost:6443 ; do sleep 1 ; done; echo Enabling the DNS in localhost; iptables -D INPUT -p udp --dport 53 -j DROP; iptables -S INPUT; echo Done"]
```

Comment:
consider installing it in the base image. Installing packages at runtime adds the risk of incompatibility, dependency, and other unknown issues

Reply:
If installing package at runtime is not a good approach, I think maybe we can directly use NCP image if we cannot find a good upstream image with ubi+iptables. I don't think it's worth to maintain a new image for IPI support.

Reply:
I also don't want to add a new image, NCP image also could work but if we use it, it seems that we need the customer to set this value manually? I'm not sure what's the best:
1. create a new customized image for IPI
2. use the official UBI image then install the package
3. use NCP image but need the user to set the image manually

Reply:
For now, the patch is using 2. @salv-orlando Do you have any comment?

Reply:
can we use operator to create this DS?